include/picongpu/param/dimension.param

```diff
@@ -31,5 +31,8 @@
 
 namespace picongpu
 {
+    /** @attention if simDim = DIM2 and using atomicPhysics, CELL_DEPTH must be set to system size integrated over in
```

Comment:
This looks atomic physics related. Should this not be catched at compile time in the code itself?